instrumentation-core/spring/autoconfigure/src/main/java/io/opentelemetry/spring/autoconfigure/trace/EnableOpenTelemetryTracing.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.opentelemetry.spring.autoconfigure.trace;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Wraps {@literal @}Configuration and {@literal @}ComponentScan(basePackages =
```

Comment:
Question: Why use @literal instead of @link since you are referring to class names already here?

e.g.,: {@link Configuration}

That's the only thing we typically use internally but I've not seen much of this.

Reply:
I used @literal to include the `@` symbol in the javadoc. Here I'm describing the functionality of the annotation. All it does is bundle two different annotations together and hardcode a specific path. It shouldn't link to another class. I'll make this more explicit.